### PR TITLE
As per Bash Style Guide

### PR DESCRIPTION
**Pull request by skthinks**

### CODE REVIEW

File: movie.sh [hunk 1]
```diff
@@ -1,58 +1,68 @@
 #!bin/bash
 
-function add_style()
-{
+mkdir tmp #Making a folder to dump files
+readonly NUM_ARGUMENTS=$#
+readonly DIRECTORY_NAME=$1
+
+function add_style(){
         tput bold
```
Comment (vishesh92) [inline movie.sh]: Keep the indentation consistent everywhere.

Comment (vishesh92) [inline movie.sh]: Its around 8 spaces here. And below its 4 spaces.

File: movie.sh [hunk 2]
```diff
@@ -1,58 +1,68 @@
 #!bin/bash
 
-function add_style()
-{
+mkdir tmp #Making a folder to dump files
```
Comment (vishesh92) [inline movie.sh]: Usually its better to leave a space after hash `#`. This makes the comments more readable.
And if you are commenting inline, then its better to leave 2 spaces before hash `#` as well.

Comment (vaidik) [inline movie.sh]: I think you missed this again? This directory is not needed. Read this http://www.tldp.org/LDP/Linux-Filesystem-Hierarchy/html/tmp.html

File: movie.sh [hunk 3]
```diff
@@ -1,58 +1,68 @@
...
+readonly NUM_ARGUMENTS=$#
+readonly DIRECTORY_NAME=$1
```
Comment (vaidik) [inline movie.sh:4]: Good one

File: movie.sh (hunk 1 above)
Comment (vaidik) [inline movie.sh]: too much indentation. Bash style guide suggests you should have only 2. I say you can do 4 as well if you like. But then it has to be consistent throughout the script. You can't have different indentation in the same script.

**Pull request merged**